lib/js/jsffi.nim

```diff
@@ -70,22 +70,30 @@ template mangleJsName(name: cstring): cstring =
   "mangledName" & $nameCounter
 
 type
-  JsRoot* = ref object of RootObj
-    ## Root type of both JsObject and JsAssoc
   JsObject* = ref object of JsRoot
     ## Dynamically typed wrapper around a JavaScript object.
   JsAssoc*[K, V] = ref object of JsRoot
     ## Statically typed wrapper around a JavaScript object.
+
   NotString = concept c
     c isnot string
-  js* = JsObject
```

Comment:
Removing ``js`` and ``JsRoot`` will cause breakage. Is this intentional? Doesn't this module depend on the definition of ``js``?

Reply:
This was requested by Araq.

Reply:
Where was this requested by Araq? On IRC? Slack?

I just asked him about it via PM and he doesn't really seem sure of the reason for this request.

Reply:
It was my impression that this shortcut alias was introduced in this PR secretly. My bad. Still I don't like this and we might want to deprecate it but that's not the point of this PR. Sorry! Please re-add it.

Reply:
It was added by myself not so long ago with other fixes and additions to `jsffi`. It's quite likely that our own project is the only user of this short-cut.

Reply:
That makes more sense. A separate PR for these things would be appreciated. The more unrelated changes are made in a single PR the harder it will be for us to figure out why the changes have been made in the future.

I thought this ``js`` global has been in ``jsffi`` since its initial inclusion. If it's something that was quickly added in a previous PR then it might be good to remove it, globals with such common names are usually a bad idea.

Reply:
We are using the shortcut quite often too :(, altho nowadays we added converters to eliminate .to almost completely its still useful for js{} etc!

Reply:
We are using the shortcut quite often too :(, altho nowadays we added converters to eliminate .to almost completely its still useful for js{} etc!

Reply:
I love this short-cut in our own code, but Araq's argument is that it violates NEP-1 and it's easy to introduce in a project with a simple

```nim
type js = JsObject
```

Reply:
Can't we just use `type Js = JsObject` then to be totally NEP-1 compatible